Spanner/src/Connection/Grpc.php

```diff
@@ -209,6 +211,11 @@ class Grpc implements ConnectionInterface
      */
     private $credentialsWrapper;
 
+    /**
+     * @var bool
+     */
+    private $larEnabled;
```

Comment:
Would like you to explore if we need to track this at the `PubSubClient` level.

Reply:
I am unable to think of a way we can track this at `Client` level. It is context/method based and we don't have both of this at the client. Open to suggestions here.

Reply:
That's OK, I just wanted to make sure you have taken that possibility into account, when you came up with this design.